src/components/Map.js

```diff
@@ -31,20 +31,16 @@ const Map = ({
   if (!isObjEmpty(mapsObj)) {
     const { map } = mapsObj;
     map.addListener("center_changed", function () {
+      const panCoords = map.getCenter();
+      const panLat = panCoords.lat();
+      const panLng = panCoords.lng();
       setUserHasPanned(true);
+      handleCenterChanged(panLat, panLng);
     });
   }
 
-  function handleCenterChanged() {
-    const center = {
-      latitude: mapProperties.center.lat,
-      longitude: mapProperties.center.lng,
-    };
-
-    if (!center.equals(mapProperties.center.lat, mapProperties.center.lng)) {
-      setMapProperties({ center });
-      setNearbyPlaces();
-    }
+  function handleCenterChanged(panLat, panLng) {
```

Comment:
I love how simple this function is and I love that you have it in `Map.js`. This is a clean encapsulation of responsibilities! 🙌

Reply:
Agree!